hpx/runtime/threads/coroutines/detail/context_generic_context.hpp

```diff
@@ -230,8 +215,11 @@ namespace hpx { namespace threads { namespace coroutines
                     (stack_size == -1) ?
                     alloc_.minimum_stacksize() : std::size_t(stack_size)
                 )
-              , stack_pointer_(alloc_.allocate(stack_size_))
+            {}
+
+            void init()
             {
+                stack_pointer_ = alloc_.allocate(stack_size_);
```

Comment:
Should there be a check here for if the stack_pointer_ has already been allocated? Or is this implementation somehow different?